site.yml

```diff
@@ -20,6 +20,10 @@
       tags: ['always']
   environment: "{{ env_vars|default({}) }}"
 
+- include: playbooks/run-openstack-ansible-security.yml
```

Comment:
I don't think we should be including playbooks from site.yml.
I'd rather see the roles called out directly here.
I think it's ok if the playbook has the same role callouts as it currently does.
It's a bit non-DRY, but we can treat site.yml as canonical